Configuration/TSconfig/ContentWizard/_wizard.ts

```diff
@@ -0,0 +1,16 @@
+# add multicolumn container to common tab
```

Comment:
Please name the file NewContentElementWizard.ts (in TSconfig folder) and include it separately in ext_localconf.php
Please remove the comment in the first line.